great_expectations/core/usage_statistics/package_dependencies.py

```diff
@@ -0,0 +1,229 @@
+        "urllib3",
+        "tzlocal",
+    ]
+
+    """This list should be kept in sync with our requirements-dev*.txt files."""
+    GE_DEV_DEPENDENCIES: List[str] = [
+        "PyMySQL",
+        "azure-identity",
+        "azure-keyvault-secrets",
+        "azure-storage-blob",
+        "black",
+        "boto3",
+        "feather-format",
+        "flake8",
+        "flask",
+        "freezegun",
+        "gcsfs",
+        "google-cloud-secret-manager",
+        "google-cloud-storage",
+        "ipywidgets",
+        "isort",
+        "mistune",
+        "moto",
+        "nbconvert",
+        "openpyxl",
+        "pre-commit",
+        "psycopg2-binary",
+        "pyarrow",
+        "pyathena",
+        "pyfakefs",
+        "pyodbc",
+        "pypd",
+        "pyspark",
+        "pytest",
+        "pytest-benchmark",
+        "pytest-cov",
+        "pytest-order",
+        "pyupgrade",
+        "requirements-parser",
+        "s3fs",
+        "snapshottest",
+        "snowflake-connector-python",
+        "snowflake-sqlalchemy",
+        "sqlalchemy",
+        "sqlalchemy-bigquery",
+        "sqlalchemy-dremio",
+        "sqlalchemy-redshift",
+        "teradatasqlalchemy",
+        "xlrd",
+    ]
+
+    def __init__(self):
+        self._requirements_relative_base_dir = "../../../"
+        self._dev_requirements_prefix: str = "requirements-dev"
+
+    def get_required_dependency_names(self) -> List[str]:
+        """Sorted list of required GE dependencies"""
+        return sorted(self.GE_REQUIRED_DEPENDENCIES)
+
+    def get_dev_dependency_names(self) -> List[str]:
+        """Sorted list of dev GE dependencies"""
+        return sorted(self.GE_DEV_DEPENDENCIES)
+
+    def get_required_dependency_names_from_requirements_file(self) -> List[str]:
+        """Get unique names of required dependencies.
+
+        Returns:
+            List of string names of required dependencies.
+        """
+        return sorted(
+            list(
+                set(
+                    self._get_dependency_names_from_requirements_file(
+                        self.required_requirements_path
+                    )
+                )
+            )
+        )
+
+    def get_dev_dependency_names_from_requirements_file(self) -> List[str]:
+        """Get unique names of dependencies from all dev requirements files.
+        Returns:
+            List of string names of dev dependencies.
+        """
+        dev_dependency_names: Set[str] = set()
+        dev_dependency_filename: str
+        for dev_dependency_filename in self.dev_requirements_paths:
+            dependency_names: List[
+                str
+            ] = self._get_dependency_names_from_requirements_file(
+                os.path.join(
+                    self._requirements_relative_base_dir, dev_dependency_filename
+                )
+            )
+            dev_dependency_names.update(dependency_names)
+        return sorted(list(dev_dependency_names))
```

Comment:
Same nit here - not a blocker but if decide to update, please apply throughout 🙇🏽

Reply:
Done!